Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/model/RecordLocatorService.java

```diff
@@ -0,0 +1,58 @@
+package gov.hhs.fha.nhinc.patientcorrelation.nhinc.model;
+
+import java.io.Serializable;
+
+/**
+ *
+ * @author ptambellini
+ */
+public class RecordLocatorService implements Serializable {
```

Comment:
Is there a specific reason that these JPAs are Serializable? We are not sending these objects over the network. Non-issue here just curious on the reasoning behind it.